lib/cli/daemonutility.cpp

```diff
@@ -145,7 +147,14 @@ bool DaemonUtility::ValidateConfigFiles(const std::vector<std::string>& configs,
 	if (!success)
 		return false;
 
+#ifdef I2_DEBUG
+	//std::cout << "Global NS: " << Serialize(ScriptGlobal::GetGlobals()) << std::endl;
+#endif /* I2_DEBUG */
```

Comment:
Also here

Reply:
That's on purpose, also the above include.

Reply:
It's dead code :skull: